cypress/integration/SEP.ts

```diff
@@ -14,7 +14,7 @@ const sepMembers = {
   reviewer2: initialDBData.users.user3,
 };
 
-function readWriteReview() {
+function readWriteReview({ shouldSubmit } = { shouldSubmit: false }) {
```

Comment:
could this be just like?
`readWriteReview(shouldSubmit: false)`

Reply:
It could be something like: `readWriteReview(shouldSubmit = false)` but my point here is to be more readable when you call the function that is why I want to have parameter as an object and not as a variable. For example I find this (`readWriteReview({ shouldSubmit: true })`) more readable then this (`readWriteReview(true)`). In the second situation you don't really know what this true is exactly doing if you don't check the definition of the function.